vendor/bat-native-ads/src/bat/ads/internal/ml/data/text_data_unittest.cc

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/data/text_data.h"
+#include <string>
+
+#include "bat/ads/internal/unittest_base.h"
+#include "bat/ads/internal/unittest_util.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+namespace ml {
+
+class BatAdsTextDataTest : public UnitTestBase {
+ protected:
+  BatAdsTextDataTest() = default;
+
+  ~BatAdsTextDataTest() override = default;
+};
+
+TEST_F(BatAdsTextDataTest, TextDataInitialization) {
+  // Arrange
+  std::string original_text = "original text";
+  data::TextData text_data(original_text);
+
+  // Act
+  std::string extracted_text = text_data.GetText();
+
+  // Assert
+  EXPECT_EQ(extracted_text, original_text);
```

Comment:
nit: would be nice to be consistent with ads tests using `expected_`, i.e. `expected_text` and `text`

Reply:
Done